js/index.js

```diff
@@ -19,8 +20,97 @@ const ProcessManager = require('./process');
 const AuthManager = require('./auth-manager');
 const WebAuth = require('./webauth');
 const unp = require('./unp-constants');
+const eureka = require('eureka-js-client').Eureka;
+const http = require('http');
+
 
 
+
+const MEDIATION_LAYER_INSTANCE_DEFAULTS = {
```

Comment:
Well, the idea behind index.js was that it wires various pieces together but doesn't contain much logic itself. Mediation layer support is worth a separate module IMO.

Reply:
Yes that would be fine, but it still needs to act upon the plugin loading events.